test/performance/odp_bench_packet.c

```diff
@@ -138,6 +151,8 @@ static void sig_handler(int signo ODP_UNUSED)
 	odp_atomic_store_u32(&gbl_args->suite.exit_worker, 1);
 }
 
+static int bench_packet_export(void *data);
```

Comment:
I think you can just define the function here directly if you need it at this point.